docs/_includes/migrate-from-msword.adoc

```diff
@@ -0,0 +1,234 @@
+// == Importing from MicroSoft Word
+:experimental:
+:url-pandoc: http://pandoc.org
+:url-ant: http://ant.apache.org/
+:url-google-asciidoc: https://chrome.google.com/webstore/detail/asciidoc-processor/eghlmnhjljbjodpeehjjcgfcjegcfbhk/
+:url-google-asciidoc-source:  https://github.com/Mogztter/asciidoc-googledocs-addon/
+
+=== Simple Pandoc
+
+Pandoc ({url-pandoc}) is a swiss army knife for converting file formats, and makes a good job of simple docx files.
+Use this command line:
+
+ $ pandoc --from=docx --to=asciidoc --wrap=none --atx-headers --normalize --extract-media=build/images input.docx > output.adoc
+
+then edit the output file to tidy it up.
+
+The conversions you can expect are shown in the table below (tested using MS Word 2010 and Windows Pandoc 1.17).
+
+.Pandoc MS Word to AsciiDoc conversions
+[cols="<20,<20"]
+|====
+|MS Word Feature |Conversion
+
+|Headings (using MS Word Heading styles 1-5)
+|Yes
+
+|Tables
+|Yes. 
+Merged cells are unmerged. 
+Column widths are ignored.
+
+|Bulleted or numbered lists
+|Yes
+
+|Footnotes
+|Yes
+
+|Figure and table captions
+|Normal paragraph
+
+|Any other MS Word styled paragraphs
+|Normal paragraph
+
+|Embedded images
+|Yes
+
+|Character formating (bold, underline and italic)
+|Yes
+
+|Document automation (fields, auto-generated figure and table numbers)
+|No - ignored
+
+|Internal references (e.g. "See Figure 3")
+|Plain text
+
+|Drawing canvas
+|Ignored
+
+|Text boxes
+|Ignored
+
+|Linked (not embedded) images
+|Ignored
+
+|Vector graphics (MS Word "insert shape")
+|Ignored
+
+|====
+
+=== Optimised Pandoc
+
+The basic usage above is fine for one-off imports.
+If you have lots to do, it is worth while cleaning the input document first, and automating the post-conversion tidy up.
+
+// Is this egg sucking? I have just told the reader what is and isn't implemented, if he can drive Word he doesn't need me to tell him what to do.
+// Are the comments helpful? Should they go into the text?

```

Comment:
I actually kind of like this section. It gives the reader the confidence and know how in making a good export. I expect that over time others will discover additional tricks that we can add here. We may also pare down the list as we find things that weren't helpful after all.